File: class05/java/AP0503-CalculadoraSimples.java
import java.util.Scanner;

public class CalculadoraSimples {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Primeiro numero: ");
        double a = sc.nextDouble();
        System.out.print("Operacao (+ - * /): ");
        String op = sc.next();
        System.out.print("Segundo numero: ");
        double b = sc.nextDouble();
        switch (op) {
            case "+": System.out.println("Resultado: " + (a + b)); break;
            case "-": System.out.println("Resultado: " + (a - b)); break;
            case "*": System.out.println("Resultado: " + (a * b)); break;
            case "/":
                if (b != 0) System.out.println("Resultado: " + (a / b));
                else System.out.println("Erro: divisao por zero");
                break;
            default: System.out.println("Operacao invalida");
        }
    }
}

File: class05/cpp/AP0503-CalculadoraSimples.cpp
#include <iostream>
int main() {
    double a, b;
    std::string op;
    std::cout << "Primeiro numero: ";
    std::cin >> a;
    std::cout << "Operacao (+ - * /): ";
    std::cin >> op;
    std::cout << "Segundo numero: ";
    std::cin >> b;
    if (op == "+") std::cout << "Resultado: " << (a + b) << std::endl;
    else if (op == "-") std::cout << "Resultado: " << (a - b) << std::endl;
    else if (op == "*") std::cout << "Resultado: " << (a * b) << std::endl;
    else if (op == "/") {
        if (b != 0) std::cout << "Resultado: " << (a / b) << std::endl;
        else std::cout << "Erro: divisao por zero" << std::endl;
    } else std::cout << "Operacao invalida" << std::endl;
    return 0;
}
